src/components/Banners/SiteBanner.tsx

```diff
@@ -29,9 +30,17 @@ export const SiteBanner = () => {
     Cookies.set(text, "viewed", { expires: 7 });
   };
 
+  const jiraLinkify = (text: string) => {
+    const JIRA_REGEXP = /[A-Z]{1,10}-\d{1,6}/ig;
+    const linkified = text.replace(JIRA_REGEXP, function(match) {
+      return '<a href="https://'+jiraHost +'/browse/' + match + '">' + match + '</a>';
+    });
+    return {__html: linkified}
+  }
+
   return showBanner ? (
     <Banner bannerTheme={theme} data-cy="sitewide-banner">
-      {text}{" "}
+      <span dangerouslySetInnerHTML={jiraLinkify(text)}></span>
```

Comment:
Can you please include a screenshot of the changes in the pr description?

Reply:
Done.